frame/babe/src/lib.rs

```diff
@@ -152,6 +152,13 @@ decl_storage! {
 		/// Temporary value (cleared at block finalization) which is `Some`
 		/// if per-block initialization has already been called for current block.
 		Initialized get(fn initialized): Option<MaybeVrf>;
+
+		/// How late the current block is compared to its parent.
+		///
+		/// This entry is populated as part of block execution and is cleaned up
+		/// on block finalization. Querying this storage entry outside of block
+		/// execution context should always yield `None`.
+		Lateness get(fn lateness): Option<T::BlockNumber>;
```

Comment:
I don't understand why this is an `Option`.

If we remove the `Option` and call `kill()` and `get()` afterwards we will get `0`.

Reply:
Yeah, updated.